arco/.htaccess

```diff
@@ -17,7 +17,11 @@ RewriteRule ^ontology/([a-zA-Z-_0-9]+)(\-aligns)(/?)$ %{ENV:ROOT_URL}/ontologie/
 RewriteCond %{HTTP_ACCEPT} ^.*text/html.*$
 RewriteRule ^ontology/([a-zA-Z-_0-9]+)(/)?$ %{ENV:LODE_URL}%{ENV:ROOT_URL}/ontologie/$1/$1.owl [R=303,L]
 
+RewriteCond %{HTTP_ACCEPT} ^.*text/html.*$
+RewriteRule ^ontology/([a-zA-Z-_0-9]+)/([0-9].[0-9])(/)?$ %{ENV:LODE_URL}%{ENV:ROOT_URL}/ontologie/$1/$2/$1.owl [R=303,L]
+
 RewriteRule ^ontology/([a-zA-Z-_0-9]+)(/)?$ %{ENV:ROOT_URL}/ontologie/$1/$1.owl [R=303,L]
+RewriteRule ^ontology/([a-zA-Z-_0-9]+)/([0-9].[0-9])(/)?$ %{ENV:ROOT_URL}/ontologie/$1/$2/$1.owl [R=303,L]
```

Comment:
Is the change two lines up doing the same match?  Seems like this line wouldn't be hit.  Similar with the line above matching an earlier line.  Also seems the html RewriteConds are mostly redundant?  Maybe I'm missing some tricks here.

Reply:
I have to admit that I'm not expert, I was trying to define a rule matching uris with a trailing number (e.g. ontology/arco/1.0).
The lines that I added match the same URIs but the first rule should be conditioned to the content type required.

Reply:
I'm no expert either. ;-)  I just try rules until things work.  And apparently I get easily confused looking at other peoples rules.  I think this is fine.  It'll use that one rule if it's html, else the owl rule.